Plan/common/src/main/java/com/djrapitops/plan/delivery/rendering/json/PlayersTableJSONCreator.java

```diff
@@ -143,82 +138,89 @@ private void appendPlayerData(StringBuilder dataJSON, TablePlayer player) {
 
         Html link = openPlayerPageInNewTab ? Html.LINK_EXTERNAL : Html.LINK;
 
-        dataJSON.append(makeDataEntry(link.create(url, StringUtils.replace(StringEscapeUtils.escapeHtml4(name), "\\", "\\\\")), "name")).append(',') // Backslashes escaped to prevent json errors
-                .append(makeDataEntry(activityIndex.getValue(), activityString, "index")).append(',')
-                .append(makeDataEntry(playtime, numberFormatters.get(FormatType.TIME_MILLISECONDS).apply(playtime), "playtime")).append(',')
-                .append(makeDataEntry(loginTimes, "sessions")).append(',')
-                .append(makeDataEntry(registered, numberFormatters.get(FormatType.DATE_YEAR).apply(registered), "registered")).append(',')
-                .append(makeDataEntry(lastSeen, numberFormatters.get(FormatType.DATE_YEAR).apply(lastSeen), "seen")).append(',')
-                .append(makeDataEntry(geolocation, "geolocation"));
+        putDataEntry(dataJson, link.create(url, StringUtils.replace(StringEscapeUtils.escapeHtml4(name), "\\", "\\\\") /* Backslashes escaped to prevent json errors */), "name");
+        putDataEntry(dataJson, activityIndex.getValue(), activityString, "index");
+        putDataEntry(dataJson, activePlaytime, numberFormatters.get(FormatType.TIME_MILLISECONDS).apply(activePlaytime), "activePlaytime");
```

Comment:
This line breaks API versioning, removing expected "playtime" field from /v1/players and /v1/network/players responses